languagetool-language-modules/crh/src/main/resources/org/languagetool/resource/crh/dev/build.gradle

```diff
@@ -0,0 +1,252 @@
+plugins {
```

Comment:
Does this work in parallel to Maven? When would it be called and by whom? (e.g. only by devs working on crh?)

Reply:
This folder is only for developing and publishing a dictionary and is not part of LT build, I'll update README